### PR TITLE
Add State Examples 

### PR DESCRIPTION
**Pull request by damassi**
Adds state examples

### CODE REVIEW

**Comment by orta** [comment]
Very cool, looks good 👍

File: 5_intro-to-state/1-basic/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Local State
+
+Example app showing how to use React's `setState` method for managing state. See [here](https://reactjs.org/docs/state-and-lifecycle.html) for more info.
+
+```
+yarn install
+yarn start
```
Comment (alloy) [inline 5_intro-to-state/1-basic/README.md]: Add instruction on how to install yarn (using npm)

File: 5_intro-to-state/2-redux/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Redux
+
+Example app showing how to use Redux for managing state. See [here](https://redux.js.org) for more info.
+
+```
+yarn install
+yarn start
```
Comment (alloy) [inline 5_intro-to-state/2-redux/README.md]: Same

File: 5_intro-to-state/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+- [React's `setState`](1-basic)
+- [Redux](2-redux)
+- [Unstated](3-unstated)
+
+In each example, run
+
+```
+yarn install
+yarn start
```
Comment (alloy) [inline 5_intro-to-state/README.md]: Same

**Pull request merged**